monero-outreach-docs/translations/es/archive/rpc_pay_es.md

```diff
@@ -1,113 +1,109 @@
 
 ### _RPC (Llamadas de procedimiento remoto)_
 
-Before studying RPC-Pay, it is helpful to understand the larger concept of RPC. RPC is a process through which one program (a client) requests data and calculations from another program (the server). The server can be on any computer on the Internet. An RPC request is like a subroutine call in a regular program except with RPC the subroutine work is performed on the server. Special intermediary software handles the network communication of both the query and its answer so that the programmer sees it as seamless. Many different types of interface software have been used to implement RPC over the years.
-Antes de estudiar sobre los Pagos RPC, es útil comprender el amplio concepto de RPC. RPC es el proceso por el cual un programa (un cliente) solícita información
+Antes de estudiar sobre qué es RPC-Pay, es útil comprender el concepto amplio de RPC. RPC es el proceso por el cual un programa (un cliente) solícita información y cálculos desde otro programa (el servidor). El servidor puede estar en cualquier computador en el Internet. Una solicitud RPC es como una llamada de subrutina en un programa normal, excepto que con el RPC, el trabajo que se ejecuta de la subrutina se realiza en el servidor. Un software intermediario específico se encarga de la red de comunicación entre la consulta y la respuesta para que el programador la va sin inconvenientes. Muchos tipos de software de interfaz han sido utilizados a través de los años para implementar llamadas RPC.
 
-A modern family of RPC interface software uses the language of JavaScript Object Notation (JSON) to form messages. JSON is a standard textual format for exchanging data. It relies on name-value pairs and lists of data to define information in a way that is robust, extendable, and human readable. JSON encoding is the primary method used by Monero’s RPC framework.
+Un grupo moderno de la familia de software de interfaz RPC utiliza el lenguaje de JavaScript de notación de objetos (JSON) para formar mensajes. JSON es un formato de texto estándar para intercambiar información. Se basa en el par "nombre-valor" y listas de datos que definen la información de una manera amplia, extendible y legible par las personas. La codificación JSON es el principal método usado por el esquema RPC de Monero.
```

Comment:
This is a suggestion to reduce the use of _**de**_:

_**Un familia actual de software de interfaz RPC...**_

This I think is a correction for the translation:

_Se basa en el par "nombre-valor" y listas de datos **para definir** información de una manera amplia, extendible y legible **para** las personas._

Reply:
Fixed and changed. :)